### ISSUE TITLE
Fine-tune with custom dataset

### ISSUE THREAD

**Issue by kime541200** [LiheYoung/Depth-Anything#35]
This is really a fantastic project, thank you for your effort! 

Will you provide methods to fine-tuning this model with custom dataset?

**Comment by LiheYoung** [comment]
We use ZoeDepth to fine-tune from the Depth Anything pre-trained encoder for metric depth estimation. You can simply follow [this example](https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth).

Actually, if you already have your own codebase, all you need is to change your model to our Depth Anything model, *i.e.*, using our pre-trained encoder and leaving the decoder randomly initialized, and then fine-tuned with your custom dataset.
